gcp_variant_transforms/testing/integration/run_preprocessor_tests.py

```diff
@@ -152,24 +123,25 @@ def _get_test_configs():
   return run_tests_common.get_configs(test_file_path, required_keys)
 
 
-def _run_test(test, context):
-  test.run(context)
+def _run_test(test):
+  # type: (PreprocessorTestCase) -> None
+  test.run_test()
+  test.validate_result()
 
 
 def main():
   """Runs the integration tests for preprocessor."""
   args = _get_args()
   test_case_configs = _get_test_configs()
-  with TestContextManager(args) as context:
-    pool = multiprocessing.Pool(processes=len(test_case_configs))
-    results = []
-    for config in test_case_configs:
-      test = PreprocessorTestCase(context, **config)
-      results.append(
-          (pool.apply_async(func=_run_test, args=(test, context)), test))
-
-    pool.close()
-    pool.join()
+
+  pool = multiprocessing.Pool(processes=len(test_case_configs))
```

Comment:
So for some reason I cannot find the old comment thread, repeating what you said and then my reply:

_"Thanks! It is a very good idea. But I prefer to change these two (preprocessor and the vcf to bq ) together because of the common module. If you're OK with it, I will remove the multiprocessing in the next PR."_

Then please either add a TODO to remove multiprocess from both tests or create a GitHub issue to track this. Either one is fine with me.